test/gen_integration.sh

```diff
@@ -1,9 +1,11 @@
+WORKFLOWS_PATH=.github/workflows
```

Comment:
💡 sharing observation/thoughts: I am not sure how this shell is consumed, seems like its getting called from another script which might have `shebang aka #!/bin/sh` also t sure the scope of the variable but `export` will let you use for the new child shell usage.

Reply:
thanks for the heads up!